README.md

```diff
@@ -2,6 +2,8 @@
 
 A project based learning activity for people who are getting started with Git and GitHub.
```

Comment:
Really great to have project-based learning activities for us who are new to Git and GitHub!